rules/jboss-eap/weblogic/WebLogicWebServiceRules.windup.xml

```diff
@@ -129,6 +130,33 @@
                 </iteration>
             </perform>
         </rule>
+        <rule id="WebLogicWebServiceRules_36ojlq">
+            <when>
+                <javaclass references="weblogic.common.T3ServicesDef" as="default">
+                    <location>TYPE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <hint message="In JBoss 5, look up the MBean replacement for the T3ServicesDef." effort="1">
```

Comment:
I think this JBoss5 rule can be removed since we are targeting EAP6 with this ruleset.